requirements.txt

```diff
@@ -6,7 +6,7 @@ cffi
 click
 cliff
 cmd2
-cryptography
+cryptography==1.9
```

Comment:
This change makes me think if we should create a document explaining all requirements.

Reply:
I think valid the documentation of requirements. I spent a lot of time figuring out what was the problem.

Reply:
Agreed.